04-onion-routing.md

```diff
@@ -227,6 +230,11 @@ The requirements ensure consistency in responding to an unexpected
 `outgoing_cltv_value`, whether it is the final node or not, to avoid
 leaking its position in the route.
 
+`resolution_format` is a single-byte identifier that specifies the format of
```

Comment:
I considered to use global feature bits here with a new scope for consistency, but decided against it because a new global feature bit would require 4 bytes. Multiplied by the number of hops, that seemed unnecessarily wasteful.